README.md

```diff
@@ -1,3 +1,227 @@
+
+1.  A [.geojson
+    file](https://github.com/dabreegster/odjitter/blob/main/data/road_network.geojson)
+    representing a transport network from which origin and destination
+    points are sampled
+
+``` bash
+head -6 data/road_network.geojson
+```
+
+    ## {
+    ## "type": "FeatureCollection",
+    ## "name": "road_network_min",
+    ## "crs": { "type": "name", "properties": { "name": "urn:ogc:def:crs:OGC:1.3:CRS84" } },
+    ## "features": [
+    ## { "type": "Feature", "properties": { "osm_id": "3468", "name": "Albyn Place", "highway": "tertiary", "waterway": null, "aerialway": null, "barrier": null, "man_made": null, "access": null, "bicycle": null, "service": null, "z_order": 4, "other_tags": "\"lit\"=>\"yes\",\"lanes\"=>\"3\",\"maxspeed\"=>\"20 mph\",\"sidewalk\"=>\"both\",\"lanes:forward\"=>\"2\",\"lanes:backward\"=>\"1\"" }, "geometry": { "type": "LineString", "coordinates": [ [ -3.207438, 55.9533584 ], [ -3.2065953, 55.9535098 ] ] } },
+
+The `jitter` function requires you to set the maximum number of trips
+for all trips in the jittered result. A value of 1 will create a line
+for every trip in the dataset, a value above the maximum number of trips
+in the ‘all’ column in the OD ata will result in a jittered dataset that
+has the same number of desire lines as in the input (50 in this case).
+
+With reference to the test data in this repo, you can run the `jitter`
+command line tool as follows:
+
+``` bash
+# With max-per-od of 50
```

Comment:
Not sure the comment explains anything that the flag and output filename doesn't :P

Reply:
:+1: